packages/blockchain-utils-validation/package.json

```diff
@@ -33,6 +33,7 @@
     "@tendermint/sig": "^0.6.0",
     "@zondax/filecoin-signing-tools": "^0.13.0",
     "caip": "^0.9.2",
+    "near-api-js": "^0.36.3",
```

Comment:
This still needs to be removed from the regular deps.

Reply:
ah, the rebase put it in both deps :/ I removed now.